prow/config/config.yaml

```
@@ -1574,10 +1574,14 @@ branch-protection:
               protect: true
               required_status_checks:
                 contexts:
-                  - "pull (verify)"
-                  - "pull (build)"
-                  - "pull (unit-test)"
-                  - "pull (integration-test)"
+                  - "ci / pull (amd64, verify) (pull_request)"
+                  - "ci / pull (amd64, build) (pull_request)"
+                  - "ci / pull (amd64, unit-test) (pull_request)"
+                  - "ci / pull (amd64, integration-test) (pull_request)"
+                  - "ci / pull (arm64, verify) (pull_request)"
+                  - "ci / pull (arm64, build) (pull_request)"
+                  - "ci / pull (arm64, unit-test) (pull_request)"
+                  - "ci / pull (arm64, integration-test) (pull_request)"
```

Remove the prefix and suffix of GitHub Action.

```suggestion
                  - "pull (amd64, verify)"
                  - "pull (amd64, build)"
                  - "pull (amd64, unit-test)"
                  - "pull (amd64, integration-test)"
                  - "pull (arm64, verify)"
                  - "pull (arm64, build)"
                  - "pull (arm64, unit-test)"
                  - "pull (arm64, integration-test)"
```